FinancialTracker/SceneDelegate.swift

```diff
@@ -68,7 +85,7 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
     func sceneDidEnterBackground(_ scene: UIScene) {
         // timeIntervalSinceNow is greater than timeActive, so i multiply by -1
         let timeActive = startDate.timeIntervalSinceNow * -1
-        FirebaseHandler.shared.addScoreToUserBasedOnTime(timeActive) { firebaseError, _ in
+        authManager.databaseManager.addScoreToUserBasedOnTime(timeActive) { firebaseError, _ in
```

Comment:
`databaseManager` should be private to `authManager`. Just expose the `databaseManager` functions through the `authManager`.

Reply:
You other options is to use them separately instead of one being hosted into the other one.